dora/minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java

```diff
@@ -149,8 +161,26 @@ protected void startProxy() throws Exception {
     }
   }
 
+  @Override
+  protected void waitForMasterServing() throws TimeoutException, InterruptedException {
+    CommonUtils.waitFor("master starts serving RPCs", () -> {
+      try (BlockMasterClient blockMasterClient = BlockMasterClient.Factory.create(
+          MasterClientContext.newBuilder(ClientContext.create(Configuration.global())).build())) {
```

Comment:
here you use global config. I wonder if this cluster can be created with some conf properties, so there's one `conf` object you should use instead of the global singleton?